packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -100,3 +126,76 @@ export const instructionToCommand = <Input, ContractInput>(instruction: Abstract
     }
   }
 }
+
+export const instructionToBatchCommand = <Input, ContractInput>(
+  instruction: AbstractInstruction<Input, ContractInput>,
+) => {
+  const id = `${instruction.instruction.contract}:${instruction.instruction.function}:batch`
+  const category = `${instruction.instruction.category}`
+  const examples = [] // TODO: Pass in accurate batch examples
+
+  return class Command extends TerraCommand {
+    static id = id
+    static category = category
+    static examples = examples
+
+    command: AbstractCommand
+
+    constructor(flags, args) {
+      super(flags, args)
+    }
+
+    buildCommand = async (flags, args): Promise<TerraCommand> => {
+      var inputs: Input[] = []
+      var contractInputs: ContractInput[] = []
+
+      flags.input.forEach(async (element) => {
```

Comment:
This will replicate what the underlying command is doing. Let's just leave the command to build itself with the input that we provide.

My idea is:
- User provide some input, via JSON file or CLI, that we could translate to a list of command inputs.
- We build N commands for N inputs using the own command builder function. This will autogenerate and validate every input
- The wrapper `makeRawTransaction` will basically compile every underlying command  `makeRawTransaction` message